helm-chart/templates/deployment.yaml

```
@@ -0,0 +1,88 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Values.controlPlane.name }}-deploy
+  namespace: {{ .Values.namespace }}
+  labels:
+    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
```

```suggestion
    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version }}
```

Helm documentation mention some recommended labels we should put on ressources:
https://helm.sh/docs/chart_best_practices/labels/#standard-labels

They also mention it to help selectors here: https://helm.sh/docs/chart_best_practices/pods/#podtemplates-should-declare-selectors